### PR TITLE
Added translation into Russian Language

### PR DESCRIPTION
**Pull request by andrey-helldar**

### CODE REVIEW

File: resources/lang/ru/passwords.php [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    | that are given by the password broker for a password update attempt
+    | has failed, such as for an invalid token or invalid new password.
+    |
+    */
+
+    'password' => 'Пароли должны быть не менее шести символов и соответствовать подтверждению.',
+    'reset' => 'Ваш пароль был сброшен!',
+    'sent' => 'Мы отправили вам по электронной почте ссылку на сброс пароля!',
+    'token' => 'Токен сброса пароля недействителен.',
+    'user' => "Мы не можем найти пользователя с этим адресом электронной почты.",
```
Comment (a-komarev) [inline resources/lang/ru/passwords.php]: Here should be `'` quotes and not `"`.

Comment (andrey-helldar) [reply resources/lang/ru/passwords.php]: See https://github.com/laravel/laravel/blob/master/resources/lang/en/passwords.php#L20

Comment (a-komarev) [reply resources/lang/ru/passwords.php]: Yes, but it's required in English variant because of `can't` word, and there is no need in it in Russian version.

**Comment by andrey-helldar** [comment]
@caouecs Manually copy the repository files?.. Doubtful pleasure.

**Comment by a-komarev** [comment]
@andrey-helldar make a PR to that repository which will automatically publish languages you need and you will be pleased.

**Comment by a-komarev** [comment]
https://github.com/caouecs/Laravel-lang/issues/749

**Pull request closed without merging**